### DISCUSSION TITLE
Is this implementation functional?

### DISCUSSION THREAD

**Discussion by mkerrig** [ymd-h/HashDL#1] [Q&A]
I tried running some of the examples + a few other datasets, and loss doesn't really appear to be going down on any reasonable settings of K, L, & sample_size. 

Also none of the examples even really seem to work? Am I missing something, or was this project abandoned because it isn't functional?

Thanks!

**Comment by ymd-h** [comment]
@mkerrig 

As you expected, I couldn't succeed yet.
The priority is not high now, but I haven't abandoned this project.
